src/app.rs

```diff
@@ -0,0 +1,349 @@
+                            } else {
+                                egui::Color32::GRAY
+                            }
+                        } else {
+                            egui::Color32::TRANSPARENT
+                        };
+                        ui.label(egui::RichText::new(text).background_color(color));
+                    });
+                }
+            },
+        );
+    }
+
+    fn render_memory_panel(debugger_panel: &mut Ui, game_data: &mut GameData) {
+        let available_width = debugger_panel.available_width();
+        egui::SidePanel::right("Memory panel")
+            .resizable(false)
+            .default_width(available_width)
+            .show_inside(debugger_panel, |ui| {
+                ui.set_width(available_width);
+
+                Self::render_registers_panel(ui, game_data);
+                Self::render_ram_panel(ui, game_data);
+
+                ui.allocate_space(ui.available_size());
+            });
+    }
+
+    fn render_registers_panel(memory_panel: &mut Ui, game_data: &mut GameData) {
+        egui::TopBottomPanel::top("Registers panel")
+            .resizable(true)
+            .default_height(memory_panel.available_height() * DEFAULT_REGISTERS_PANEL_HEIGHT_RATIO)
+            .show_inside(memory_panel, |ui| {
+                ui.style_mut().wrap_mode = Some(egui::TextWrapMode::Extend);
+
+                ui.label("Registers");
+                ui.add(egui::Separator::default().horizontal());
+
+                egui::ScrollArea::both().show(ui, |ui| {
+                    Self::render_cpu_registers(ui, &mut game_data.gameboy);
+                });
+            });
+    }
+
+    fn render_cpu_registers(registers_panel: &mut Ui, gameboy: &mut Gameboy) {
+        registers_panel.label(format!(
+            "a: {:02X}\t\tb: {:02X}",
+            gameboy.cpu.registers.a, gameboy.cpu.registers.b
+        ));
+        registers_panel.label(format!(
+            "c: {:02X}\t\td: {:02X}",
+            gameboy.cpu.registers.c, gameboy.cpu.registers.d
+        ));
+        registers_panel.label(format!(
+            "e: {:02X}\t\tf: {:02X}",
+            gameboy.cpu.registers.e, gameboy.cpu.registers.f
+        ));
+        registers_panel.label(format!(
+            "h: {:02X}\t\tl: {:02X}",
+            gameboy.cpu.registers.h, gameboy.cpu.registers.l
+        ));
+
+        registers_panel.label(format!(
+            "pc: {:04X}\tsp: {:04X}",
+            gameboy.cpu.registers.pc, gameboy.cpu.registers.sp
+        ));
+        registers_panel.label(format!("ime: {}", gameboy.cpu.ime));
+    }
+
+    fn render_ram_panel(memory_panel: &mut Ui, game_data: &mut GameData) {
+        let available_height = memory_panel.available_height();
+        egui::TopBottomPanel::bottom("RAM panel")
+            .resizable(false)
+            .default_height(available_height)
+            .show_inside(memory_panel, |ui| {
+                ui.set_height(available_height);
+
+                ui.label("RAM:");
```

Comment:
It's more about memory map than RAM

Reply:
Done: https://github.com/hferraud/gbmu/pull/7/commits/fe7fba87250dcd82970d443321e63a7a65d7b224